akka-docs/rst/scala/stream/migration-guide-2.0-2.4-scala.rst

```diff
@@ -79,6 +79,17 @@ In Akka 2.4.x this is formulated like so:
 
 .. includecode:: ../code/docs/stream/MigrationsScala.scala#expand-state
 
+Changes in Akka HTTP
+====================
+
+Routing settings parameter name
+-------------------------------
+
+``RoutingSettings`` were previously the only setting available on ``RequestContext``,
+and were accessible via ``settings``. We now made it possible to configure the parsers
+settings as well, so ``RoutingSettings`` is now ``routingSettings`` and ``ParserSetttings`` is
+now accessible via ``parserSettings``.
```

Comment:
Butbut, looking at `RequestContext` in javadsl I don't see any `RoutingSetting` s?